pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -156,9 +156,11 @@ stdenv.mkDerivation {
       wrap "$ldInner" ${./macos-sierra-reexport-hack.bash} ''${ld:-$ldPath/${targetPrefix}ld}
       wrap "${targetPrefix}ld" ${./ld-wrapper.sh} "$out/bin/$ldInner"
       unset ldInner
-    '') + ''
+    '')
+    # TODO make list below easier to customize without mass rebuild
+    + ''
 
-      for variant in ld.gold ld.bfd ld.lld; do
+      for variant in ld.gold ld.bfd ld.lld${optionalString (targetPlatform.useLLVM or false) " ld64.ldd"}; do
```

Comment:
I think this wasn’t actually necessary… let’s leave it out for now (perhaps later we can get macOS llvm cross working)

Reply:
Yeah it's not necessary; I was just too busy fixing other things to move it out yet.